landing/public/css/home.css

```diff
@@ -2,151 +2,519 @@ html {
+}
+
+section {
+  z-index: 1;
+}
+
+.primary-white-btn {
+  background: #fff;
+  box-shadow: var(--regular-box-shadow);
+  transition: background 200ms;
+
+}
+
+.primary-white-btn:hover {
+  background: var(--primary-main);
+  box-shadow: var(--regular-box-shadow);
+  transition: background 200ms;
+
+}
+
+.primary-white-btn-link {
+  font-weight: 700;
+  color: var(--primary-main);
+}
+
+.primary-white-btn-link:hover {
+  font-weight: 700;
+  color: #fff;
+}
+
+.heroBG {
+  position: absolute;
+  z-index: -4;
+  mix-blend-mode: multiply;
+  width: 100%;
+  height: auto;
+  max-height: 100vh;
+  opacity: 70%;
+}
+
+.heroCircle1 {
+  position: absolute;
+  top: 8px;
+  z-index: -5;
+  max-width: 80px;
+}
+
+.heroCircle2 {
+  position: absolute;
+  top: 136px;
+  left: 472px;
+  z-index: -5;
+  max-width: 160px;
 }
 
-.landing {
+.heroCircle3 {
+  position: absolute;
+  top: 584px;
+  right: 0px;
+  z-index: -5;
+  max-width: 96px;
+}
+
+.hero {
   width: 100vw;
   height: 100vh;
+  background-image: url("../static/background/heroBG.png");
+  background-blend-mode: multiply;
+  background-size: 100% 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  overflow: hidden;
+  z-index: 10;
 }
 
-.home {
-  width: 100%;
-  height: 100%;
+.heroText {
   display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: start;
+  margin-left: 120px;
+  max-width: 400px;
+}
+
+.heroHead {
+  font-size: 32px;
+  line-height: 48px;
+  color: #fff;
+  font-weight: 700;
+}
+
+.heroBody {
+  font-size: 16px;
+  line-height: 26px;
+  color: #fff;
+  margin-top: 16px;
+  margin-bottom: 32px;
+}
+
+.heroImage {
+  max-width: 50%;
+  margin-right: 24px;
+}
+
+.vp1 {
+  display: flex;
+  flex-direction: column;
+  margin: 80px 120px 160px 120px;
+}
+
+.textArea {
+  display: flex;
+  flex-direction: row;
+}
+
+.line {
+  width: 4px;
+  height: auto;
+  background-color: var(--secondary-main);
+  margin-right: 16px;
+}
+
+.vpText {
+  display: flex;
+  flex-direction: column;
+  max-width: 75%;
+}
+
+.vpText h1{
+  font-weight: 700;
+}
+
+.vpText p{
+  margin-top: 8px;
+  color: var(--muted-default);
+}
+
+.vp1Image {
+  margin-top: 40px;
+}
+
+.circleBack1 {
+  max-width: 200px;
+  position: absolute;
+  z-index: -1;
+  top: 1480px;
+}
+
+.circleBack2 {
+  max-width: 184px;
+  position: absolute;
+  z-index: -2;
+  top: 2200px;
+  right: 0px;
+}
+
+.vp2Flex {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
   justify-content: center;
+  margin: 160px 120px;
+}
+
+.vp2 {
+  display: flex;
+  flex-direction: row;
   align-items: center;
-  align-content: center;
+  justify-content: center;
 }
 
-.home-container {
-  position: absolute;
-  animation: loading 2s forwards;
+.vp2Area {
+  max-width: 80%;
+}
+
+.vp2Image {
+  max-width: 25%;
+  margin-right: 40px;
+}
+
+.payMethods {
+  margin: 40px 0px;
+  display: flex;
+  flex-direction: column;
+}
+
+.payMethods h3 {
```

Comment:
same as the previous comment, simply add the class `extra-bold` for `.payMethods h3` instead of calling in css